### PR TITLE
[MRG+1] Include tests for non-string items to Exporters

### PR DESCRIPTION
**Pull request by stummjr**
As suggested [here](https://github.com/scrapy/scrapy/pull/1737#issuecomment-175819639), this PR adds tests for the JSON, JSON-Lines, Pickle and Marshall item exporters, regarding to items with non-string fields.

### CODE REVIEW

File: tests/test_exporters.py [hunk 1]
```diff
@@ -135,14 +144,7 @@ def test_export_binary(self):
         self.assertEqual(expected, exporter.export_item(value))
 
     def test_other_python_types_item(self):
```
Comment (eliasdorneles) [inline tests/test_exporters.py]: a nitpicking: how about renaming the methods to `test_nonstring_types_item` to match the item description?

Comment (eliasdorneles) [inline tests/test_exporters.py]: all the rest looks good to me. :+1:

**Comment by stummjr** [comment]
Hey, @eliasdorneles ! I updated the methods names, as suggested.

**Comment by eliasdorneles** [comment]
Cool, LGTM

**Pull request merged**